dashboard/utils.py

```diff
@@ -336,6 +341,11 @@ def notified_by(brother):
 
 # zips a list of tuples: (event, attendance status)
 def create_attendance_list(events, excuses_pending, excuses_approved, brother):
+    """ zips together a list of tuples where the first element is each event and the second is the brother's
```

Comment:
generally you also want to explicitly describe all the parameters.  up to you if you think thats valuable